addons/OPT/dynmarkerverwaltung/fn_clientInit.sqf

```diff
@@ -87,32 +81,105 @@ GVAR(dialogCheck) =
 	
 ] call CFUNC(addPerFrameHandler);
 
-//Initialisierung
+//alte Marker löschen
+DFUNC(markerloeschen) = 
```

Comment:
Solche Funktionen sollten als eigene Datei erstellt werden und in die ` modules.hpp`  eingetragen werden